scenarios/workflow_with_many_actions.go

```diff
@@ -0,0 +1,46 @@
+package scenarios
+
+import (
+	"context"
+
+	"github.com/temporalio/omes/loadgen"
+	"github.com/temporalio/omes/loadgen/kitchensink"
+)
+
+func init() {
+	loadgen.MustRegisterScenario(loadgen.Scenario{
+		Description: "Each iteration executes a single workflow with a number of child workflows and/or activities. " +
+			"Additional options: children-per-workflow (default 30), activities-per-workflow (default 30).",
+		Executor: loadgen.KitchenSinkExecutor{
+			PrepareWorkflowParams: func(ctx context.Context, opts loadgen.RunOptions, params *kitchensink.WorkflowParams) error {
+				// We want these executed concurrently
+				params.ActionSet.Concurrent = true
+
+				// Get options
+				children := opts.ScenarioOptionInt("children-per-workflow", 30)
+				activities := opts.ScenarioOptionInt("activities-per-workflow", 30)
```

Comment:
Would be nice if the options were more structured so they can be shown to a user describing the scenario.

Reply:
Not blocking